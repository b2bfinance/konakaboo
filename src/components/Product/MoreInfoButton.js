import styled from 'styled-components';
import Link from '@material-ui/core/Link';

export default styled(Link)`
  margin-bottom: 8px !important;
  color: ${props => props.theme.infoButtonColor} !important;
`;
