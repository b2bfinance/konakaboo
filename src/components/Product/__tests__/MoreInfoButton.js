import React from 'react';
import { render } from 'enzyme';
import MoreInfoButton from '../MoreInfoButton';

describe('MoreInfoButton', () => {
  test('renders correctly', () => {
    expect(
      render(<MoreInfoButton>more info button</MoreInfoButton>)
    ).toMatchSnapshot();
  });
});
