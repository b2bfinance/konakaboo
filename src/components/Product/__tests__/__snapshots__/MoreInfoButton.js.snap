// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreInfoButton renders correctly 1`] = `
.c0 {
  margin-bottom: 8px !important;
  color: !important;
}

<a
  class="MuiTypography-root-6 MuiTypography-colorPrimary-36 MuiTypography-inline-41 MuiLink-root-1 MuiLink-underlineHover-3 c0"
>
  more info button
</a>
`;
